packages/botonic-api/src/data-provider/local-data-provider.ts

```diff
@@ -22,143 +22,117 @@ export class LocalDevDataProvider implements DataProvider {
 
   getUsers(limit = 10, offset = 0): User[] {
     const path = this.paths.USERS
-    try {
-      this.db.reload()
-      const userList = this.db.getObject<Record<string, User>>(path)
-      const users = Object.values(userList)
-      const from = offset * limit
-      const to = limit !== 0 ? from + limit : undefined
-      return users.slice(from, to)
-    } catch (e) {
-      console.error('Error fetching users from local DB', e)
-    }
-    return []
+    this.db.reload()
```

Comment:
The local DB used is a JSON file, and since it can be updated from differents sides (websockets, api) and we have different instances for each entry point, if we don't reload on every call we will use the data loaded in memory instead of using the data from the JSON file (which could have been changed from other side) and we could end up missing some entries.